### PR TITLE
`outline-calendar-dropdown-link` Component

### PR DESCRIPTION
**Pull request by JBamheedor**
Adding new component which provides 'add to calendar' functionality leveraging [[calendar-link]]((https://github.com/AnandChowdhary/calendar-link))
Calendar Link Dropdown - Calendar Link Dropdown ⋅ Storybook 2021-11-19 11-26-44

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/142"></a>

### CODE REVIEW

**Review by mabry1985 — APPROVED** [review]

File: src/components/base/outline-calendar-link-dropdown/outline-calendar-link-dropdown.ts [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    };
+  }
+
+  render(): TemplateResult | undefined {
+    const iconName = this.open ? 'chevron-up' : 'chevron-down';
+    const menuState = this.open ? 'open' : 'hidden';
+
+    return html`
+      <div class="button-wrapper ${menuState}">
+        <button
```
Comment (Jwaxo) [inline src/components/base/outline-calendar-link-dropdown/outline-calendar-link-dropdown.ts:114]: I'm a bit new to Outline and the general modus operandi here, but would it make sense to use `outline-button` here, which also includes the option to add icons? Or is it the fact that it changes behavior on opening and clicking that causes some problems? If it were possible to use that component, we'd be able to cut down on possible duplication for styles, etc.

**Review by Jwaxo — COMMENTED** [review]
Looks great, I just have one question that may or may not be a blocker. Also looks like you have merge conflicts which you'll need to resolve now.

**Comment by damontgomery** [comment]
From this discussion:

Requirements to merge
- Test on local / gitpod

Nice to have
- Look at using outline features like outline-button, outline-accordion

**Comment by himerus** [comment]
I'm thinking an additional update here could get this merged once #261 is complete and merged.

**Pull request closed without merging**